### DISCUSSION TITLE
Commander Pro, does it save the profile set via fancontrol?

### DISCUSSION THREAD

**Discussion by laleeroy** [EvanMulawski/FanControl.CorsairLink#77] [Q&A] [answered]
Anyone have tried to use the fancontrol feature which can substract the value of 2 different temp sensors in Commander Pro?
Can it store the value applied in fancontrol into the commander pro? Would like to use mine in external water cooling setup

**Comment by EvanMulawski** [comment, marked as answer]
A fan curve can only be stored on the device using iCUE and the stored curve must use a temperature probe on the device.
